[2021-12-23 17:31:04,796] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T17:30:00+00:00 [queued]>
[2021-12-23 17:31:04,809] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T17:30:00+00:00 [queued]>
[2021-12-23 17:31:04,809] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 17:31:04,810] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-23 17:31:04,810] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 17:31:04,829] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): echo_meet_date> on 2021-12-23T17:30:00+00:00
[2021-12-23 17:31:04,830] {standard_task_runner.py:52} INFO - Started process 363853 to run task
[2021-12-23 17:31:04,840] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'echo_meet_date', '2021-12-23T17:30:00+00:00', '--job-id', '1807', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp6uhpk23i', '--error-file', '/tmp/tmp6t9_oedm']
[2021-12-23 17:31:04,844] {standard_task_runner.py:77} INFO - Job 1807: Subtask echo_meet_date
[2021-12-23 17:31:04,911] {logging_mixin.py:104} INFO - Running <TaskInstance: test.echo_meet_date 2021-12-23T17:30:00+00:00 [running]> on host airflowvm
[2021-12-23 17:31:05,108] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=echo_meet_date
AIRFLOW_CTX_EXECUTION_DATE=2021-12-23T17:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-23T17:30:00+00:00
[2021-12-23 17:31:05,108] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-23 17:31:05,109] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo 211228']
[2021-12-23 17:31:05,120] {subprocess.py:75} INFO - Output:
[2021-12-23 17:31:05,123] {subprocess.py:79} INFO - 211228
[2021-12-23 17:31:05,124] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-23 17:31:05,156] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=test, task_id=echo_meet_date, execution_date=20211223T173000, start_date=20211223T173104, end_date=20211223T173105
[2021-12-23 17:31:05,194] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-23 17:31:05,214] {local_task_job.py:151} INFO - Task exited with return code 0
